lib/gettext-setup/pot.rb

```diff
@@ -137,5 +137,19 @@ def self.update_pot(locales_path = GettextSetup.locales_path, path = nil)
         end
       end
     end
+
+    def self.merge_pot(locales_path = GettextSetup.locales_path)
+      mergepot = GettextSetup.config['project_name'] + '.pot'
```

Comment:
Yep! The reason is that our result will get consumed by the forge, which is looking specifically for projectname.pot. By calling it this by default we save ourselves the time of having to rename it or change other functionality. However I'll change this so that the possibility of renaming is there.